spec/std/isa/csr/Ssaia/stopi.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) Kallal Mukherjee.
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+
+# yaml-language-server: $schema=../../../schemas/csr_schema.json
+
+$schema: "csr_schema.json#"
+kind: csr
+name: stopi
+long_name: Supervisor Top Interrupt
+address: 0x15B
+writable: true
+priv_mode: S
+length: SXLEN
+definedBy: Ssaia
+description: |
+  Supervisor-level CSR stopi is read-only with width SXLEN. A read of stopi 
+  returns information about the highest-priority pending-and-enabled interrupt 
+  for supervisor level, in this format:
+  - bits 27:16 IID
+  - bits 7:0 IPRIO
+  All other bits of stopi are reserved and read as zeros.
+fields:
+  IID:
+    location: 27-16
+    long_name: Interrupt Identity
+    description: |
+      Interrupt identity of the highest-priority pending-and-enabled interrupt.
+    type: RW
```

Comment:
If the CSR is read-only, then this field should be read-only. However, since it does change, it must be that the hardware magically updates it, so this should be:
```
type: RO-H
```